src/main/java/org/mockito/internal/junit/DefaultStubbingLookupListener.java

```diff
@@ -20,6 +20,7 @@
 class DefaultStubbingLookupListener implements StubbingLookupListener {
 
     Strictness currentStrictness;
+    boolean mismatchesReported;
```

Comment:
>Actually having the field public (package visible) made me wonder if anyone could change the state

This is the intent of this code. This state is mutable. The alternative is to provide getter and setter, which is more code for a simple collaboration between classes in the same package. If you feel that getter and setter yield code that is easier to understand, I'm happy to change it. LMK!